pallets/file-system/src/lib.rs

```diff
@@ -819,6 +820,7 @@ pub mod pallet {
                 msp_id,
                 bucket_id,
                 name,
+                root: <T::ProofDealer as shp_traits::ProofsDealerInterface>::MerkleHash::default(),
```

Comment:
If everytime there's a new bucket, it's root is the default, why would you need to pass that information on the event? You can access this type from the client, and get its default.

Reply:
I know, I specifically wanted to make it fool-proof for someone that might want to write an indexer in another language - this way I don't have to care about which hashing scheme runtime is using - I already have all the info.